core/sup/Makefile

```diff
@@ -1,9 +1,15 @@
 ROOT = ../..
 PROJECT = sup
 
-all: sup
+all: ebin/sup.app sup
+
+ebin/sup.app:
+	@$(REBAR) compile
 
 sup:
-	@$(REBAR) compile escriptize
+	@$(REBAR) escriptize
+
+clean:
+	rm ./sup
```

Comment:
I have opened PRs for this. Please revert (and this has nothing to do in this PR)